hbase-backup/src/main/java/org/apache/hadoop/hbase/backup/RestoreJob.java

```diff
@@ -30,6 +30,9 @@
 
 @InterfaceAudience.Private
 public interface RestoreJob extends Configurable {
+
+  String KEEP_ORIGINAL_SPLITS_OPT = "hbase.backup.restorejob.keep_original_splits";
```

Comment:
nit: for configuration points that we interact with primarily through the `Configuration` objects, we usually call these variables `FOO_KEY`, not `FOO_OPT`.